src/client/SConscript

```diff
@@ -1,14 +1,19 @@
 """Build DAOS client"""
+Import('prereqs')
 
 def scons():
     """Execute build"""
 
+    if not (prereqs.client_requested() or prereqs.server_requested()):
```

Comment:
Does this make it not possible to request client and server (for the case where you just do not want test) in a build?  Maybe I am creeping the scope unnecessarily.

Reply:
You can do scons client server install